compiler/modules/nback/build.bal

```diff
@@ -212,7 +212,7 @@ class Scaffold {
     }
 }
 
-function buildModule(bir:Module mod, llvm:Context context) returns llvm:Module|BuildError {
+function buildModule(bir:Module mod, llvm:Context context, string? gcStrategy) returns llvm:Module|BuildError {
```

Comment:
This won't scale.  We should have an Options record in nback.

Also call it gcName, which is what LLVM calls it.